testutils/flag_config.go

```diff
@@ -0,0 +1,257 @@
+package testutils
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/Layr-Labs/eigenda-proxy/config"
+	"github.com/Layr-Labs/eigenda-proxy/config/eigendaflags"
+	eigendaflagsv2 "github.com/Layr-Labs/eigenda-proxy/config/v2/eigendaflags"
+	"github.com/Layr-Labs/eigenda-proxy/store"
+	"github.com/Layr-Labs/eigenda-proxy/store/generated_key/memstore"
+	"github.com/Layr-Labs/eigenda-proxy/store/precomputed_key/redis"
+	"github.com/Layr-Labs/eigenda-proxy/store/precomputed_key/s3"
+	"github.com/Layr-Labs/eigenda-proxy/verify"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	privateKeyEnvVar = "SIGNER_PRIVATE_KEY"
+	ethRPCEnvVar     = "ETHEREUM_RPC"
+	transport        = "http"
+	host             = "127.0.0.1"
+	disperserPort    = "443"
+
+	disperserPreprodHostname   = "disperser-preprod-holesky.eigenda.xyz"
+	preprodCertVerifierAddress = "0xd973fA62E22BC2779F8489258F040C0344B03C21"
+	preprodSvcManagerAddress   = "0x54A03db2784E3D0aCC08344D05385d0b62d4F432"
+
+	disperserTestnetHostname   = "disperser-testnet-holesky.eigenda.xyz"
+	testnetCertVerifierAddress = "0xFe52fE1940858DCb6e12153E2104aD0fDFbE1162"
+	testnetSvcManagerAddress   = "0xD4A7E1Bd8015057293f0D0A557088c286942e84b"
+)
+
+// FlagConfig represents an individual flag configuration, with a flag name and value
+type FlagConfig struct {
+	Name  string
+	Value string
+}
```

Comment:
not a big of this approach and feel like it was more intuitive / less verbose with what we had before. This is also rather difficult to understand vs just doing straight config injection

Reply:
I think there are some good reasons to go this route, though it seems I'm probably in the minority here.

Let's discuss in today's standup, I can revert this without too much pain, if that's the way we'd like to go